include/proxy-wasm/wasm.h

```diff
@@ -27,6 +27,7 @@
 #include "include/proxy-wasm/context.h"
 #include "include/proxy-wasm/exports.h"
 #include "include/proxy-wasm/wasm_vm.h"
+#include "src/vm_id_handle.h"
```

Comment:
You shouldn't include `src/` from `include/`. Maybe just move that file to `include/`?